rules/max-asserts.js

```diff
@@ -3,11 +3,15 @@ const {visitIf} = require('enhance-visitors');
 const util = require('../util');
 const createAvaRule = require('../create-ava-rule');
 
+const MAX_ASSERTIONS_DEFAULT = 5;
+
 const notAssertionMethods = ['plan', 'end'];
 
 const create = context => {
 	const ava = createAvaRule();
-	const maxAssertions = context.options[0] || 5;
+	// TODO: Remove default value once defaults can be applied to integers
```

Comment:
I meant a TODO about converting to a object option and then removing the fallback.